<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="375323" started-at="2021-05-13T08:19:13Z" finished-at="2021-05-13T08:25:28Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="375323" started-at="2021-05-13T08:19:13Z" finished-at="2021-05-13T08:25:28Z">
      <class name="testcases.Payment_TestCase">
        <test-method status="PASS" signature="setUp()[pri:0, instance:testcases.Payment_TestCase@7ee8290b]" name="setUp" is-config="true" duration-ms="4652" started-at="2021-05-13T13:49:08Z" finished-at="2021-05-13T13:49:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="urlSetUp()[pri:0, instance:testcases.Payment_TestCase@7ee8290b]" name="urlSetUp" is-config="true" duration-ms="4453" started-at="2021-05-13T13:49:13Z" finished-at="2021-05-13T13:49:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- urlSetUp -->
        <test-method status="PASS" signature="homePageSetUp()[pri:0, instance:testcases.Payment_TestCase@7ee8290b]" name="homePageSetUp" is-config="true" duration-ms="9743" started-at="2021-05-13T13:49:18Z" finished-at="2021-05-13T13:49:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- homePageSetUp -->
        <test-method status="FAIL" signature="validatePaymentSection()[pri:0, instance:testcases.Payment_TestCase@7ee8290b]" name="validatePaymentSection" duration-ms="360223" started-at="2021-05-13T13:49:27Z" finished-at="2021-05-13T13:55:27Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertFalse(Assert.java:63)
	at org.testng.Assert.assertFalse(Assert.java:73)
	at testcases.Payment_TestCase.validationForReportOnlinePurchase(Payment_TestCase.java:271)
	at testcases.Payment_TestCase.validatePaymentSection(Payment_TestCase.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validatePaymentSection -->
        <test-method status="PASS" signature="geneareReport(org.testng.ITestResult)[pri:0, instance:testcases.Payment_TestCase@7ee8290b]" name="geneareReport" is-config="true" duration-ms="320" started-at="2021-05-13T13:55:27Z" finished-at="2021-05-13T13:55:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=validatePaymentSection status=FAILURE method=Payment_TestCase.validatePaymentSection()[pri:0, instance:testcases.Payment_TestCase@7ee8290b] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- geneareReport -->
        <test-method status="PASS" signature="logOut()[pri:0, instance:testcases.Payment_TestCase@7ee8290b]" name="logOut" is-config="true" duration-ms="534" started-at="2021-05-13T13:55:28Z" finished-at="2021-05-13T13:55:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logOut -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:testcases.Payment_TestCase@7ee8290b]" name="closeBrowser" is-config="true" duration-ms="1521" started-at="2021-05-13T13:55:28Z" finished-at="2021-05-13T13:55:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- testcases.Payment_TestCase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
